app/controllers/admin/topics_controller.rb

```diff
@@ -366,6 +366,49 @@ def split_topic
     end
   end
 
+  def merge_tickets
+    @topics = Topic.where(id: params[:topic_ids])
+
+    @topic = Topic.new(
+      name: "MERGED: #{@topics.first.name}",
+      user: @topics.first.user,
+      forum_id: @topics.first.forum_id,
+      private: @topics.first.private,
+    )
+
+    @posts = @topic.posts
+    topics_merged = ''
+
+    if @topic.save
+      @topics.each do |t|
+        logger.info("Id #{t.id}")
+        t.posts.each do |p|
+          @posts.create(
+          body: p.body,
+          kind: p.kind,
+          user: p.user,
+          screenshots: p.screenshots,
+          attachments: p.attachments,
+          )
+        end
+
+        topics_merged << "%#{t.id}-%#{t.name}\n"
```

Comment:
You don't actually need the % signs, that was actually part of the token in the example I gave.